Mage.Server.Plugins/Mage.Player.AI.MA/src/mage/player/ai/SimulatedPlayer2.java

```diff
@@ -120,7 +120,7 @@ protected void simulateOptions(Game game) {
     }
 
     @Override
-    protected void addVariableXOptions(List<Ability> options, Ability ability, int targetNum, Game game) {
```

Comment:
There is interesting todo: "Support modal spells with more than one selectable mode", maybe `targetNum` needs here for that todo
![shot_230715_105336](https://github.com/magefree/mage/assets/8344157/a738fe5a-4c48-42e8-95e2-ef7654334c1e)